config/config.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	RunDDL       bool   `toml:"run-ddl" json:"run-ddl"`
 	SplitTable   bool   `toml:"split-table" json:"split-table"`
 	TokenLimit   int    `toml:"token-limit" json:"token-limit"`
+	DisableChunk bool   `toml:"disable-chunk" json:"disable-chunk"`
```

Comment:
s/DisableChunk/EnableChunk

Reply:
DisableChunk is better. If we do not set it, it should be false.